app/src/main/java/org/listenbrainz/android/ui/screens/feed/FeedScreen.kt

```diff
@@ -434,12 +479,12 @@ private fun MyFeed(
     onPlay: (FeedEvent) -> Unit,
     goToUserPage: (String) -> Unit,
     goToArtistPage: (String) -> Unit,
+    shimmerInstance: Shimmer,
     uriHandler: UriHandler = LocalUriHandler.current
 ) {
-    // Since, at most one drop down will be active at a time, then we only need to maintain one state variable.
```

Comment:
Try to maintain existing comments. Here, we can get rid of this as code makes the intent clear anyways.